### PR TITLE
Add voters from new data pull, exceptions

### LINKED ISSUE #5059: Changes to the voter rolls for 2020 Steering election
This year we have had several questions regarding contributors who should have been in voters.md, but were not.  Because this year’s EC includes someone who does databases, we’ve probed the missing names more than we have in the past, and this year has also involved the list of Org Members for the first time.  This has yielded some irregularities and changes, a few of which the EC needs to vote on.

August 2: Lukasz pulled original list of “over 50 contributors” (herafter 50contribs) using the same query used for the last two elections.  This query relies of a yearly snapshot of contributions which is automatically updated daily as part of the Devstats charts.

August 3: Josh pulled list of current org members from kubernetes/org repository.

August 11: compared list of 50contribs with list of org members, resulting in 620 names for voters.md.  Published list.

August 13: based on reports from kubernetes-dev, discovered that there are github handle capitalization mismatches between kubernetes/org and Devstats.  Re-ran list, adding 68 contributors to voters.md.  However, several mysteriously missing names remained missing.

August 18: since it’s not possible to re-run the report from August 1 (the snapshot has been changed), Lukasz ran a new and much simpler query to find the 50contribs.  This simpler query found 229 additional 50contribs, 84 of whom are org members.  The majority of these appear to be because of (a) accidentally splitting contributions across employers, (b) last-minute contributions, and (c) contributors who don’t have emails being excluded. 

Mysteriously, though, 195 handles which were on the original 50contribs list are not on the new list.  104 of these are org members. Because it is not possible to re-run the original report, we cannot ever be certain why.  On a spot check, several of them are old contributors whose contributions have fallen off over the last year, so for some it’s a timing issue, but that wouldn’t account for 195 differences.

Votes required of the Election Committee

1. To add the 84 new qualified handles to voters.md.
2. To keep the 104 newly disqualified handles in voters.md without removal (since we already told them they could vote)
3. To add a section to the bottom of voters.md for “Contributors whose emails we don’t have” and solicit folks to give us their email via the exception form.

/assign jdumars idvoretskyi 
/sig contributor-experience
/committee steering
/area community-management

### PR DESCRIPTION
**Pull request by jberkus**
Add 84 voters per Issue #5059 
Add voters per exceptions requests
Includes new section on voters with no email

Fixes #5059

### CODE REVIEW

**Comment by jdumars** [comment]
/hold
/lgtm

unhold once Dims is happy :)

File: events/elections/2020/voters.md [hunk 1]
```diff
@@ -155,18 +192,23 @@ The eligible voter process can be found in the [2020 elections folder].
...
 - dekkagaijin
 - denkensk
 - derekwaynecarr
+- desaintmartin
 - detiber
 - devigned
+- devlware
 - dhellmann
 - dholbach
 - dims
```
[the remark this answers is not in the document]
Comment (liggitt) [reply events/elections/2020/voters.md:205]: he means "dims" is included twice (line 205, 206)

[the remark this answers is not in the document]
Comment (jberkus) [reply events/elections/2020/voters.md:205]: Oh!  GH line quoting was confusing me.

Will fix, thanks!

[the remark this answers is not in the document]
Comment (jberkus) [reply events/elections/2020/voters.md:200]: Fixed with last force push.

**Comment by jberkus** [comment]
Fixed GH handle duplicates in file.

Also added one voter per exception request.

@jdumars @idvoretskyi please approve

**Review by idvoretskyi — APPROVED** [review]
/lgtm

/hold cancel

**Pull request merged**